python/cuml/dask/common/base.py

```diff
@@ -18,7 +18,8 @@
 import numpy as np
 from toolz import first
 
-import cudf.comm.serialize  # noqa: F401
+# TODO: Check if below import is needed
+# import cudf.comm.serialize  # noqa: F401
```

Comment:
I am not sure why this import is needed. I noticed it creates `TypeError: No dispatch for <class 'cudf.core.dataframe.DataFrame'>` to be thrown when using dask_cudf.DataFrame as input of DistributedDataHandler. Of course commenting out this import is probably not the correct solution, I'll have to figure this out before merging.

Reply:
Actually it looks like the issue was solved in a recent PR because I was not able to reproduce it. I added the import back.